samples/KvsAudioVideoStreamingSample.c

```diff
@@ -195,7 +198,7 @@ INT32 main(INT32 argc, CHAR* argv[])
     STRNCPY(videoCodec, VIDEO_CODEC_NAME_H264, STRLEN(VIDEO_CODEC_NAME_H264)); // h264 video by default
 
     if (argc == 7) {
-        if (STRCMP(argv[6], "1")) {
+        if (!STRCMP(argv[6], "1")) {
```

Comment:
Was this ! intended?

Reply:
Yes. It is. The sample was broken earlier.